runners/lock_helper.py

```diff
@@ -9,9 +9,8 @@
 
 Or it can simply be something that requires exclusive access.
 
-It assumes an environment variable called ZOOKEEPER_HOSTS, which is a comma
-separated string.
-
+The zookeeper hosts can either be passed in as a list, or as an environment variable called
+ZOOKEEPER_HOSTS, which is a comma separated string.
```

Comment:
I think it's not very pythonic to pass in arguments via implicit env vars (explicit > implicit), so we should probably completely sunset env vars since we're already shifting away from them. :dancer: